Geometry/HGCalCommonData/src/HGCalCellOffset.cc

```diff
@@ -0,0 +1,138 @@
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "Geometry/HGCalCommonData/interface/HGCalCellOffset.h"
+#include <vector>
+#include <iostream>
+
+//#define EDM_ML_DEBUG
+
+HGCalCellOffset::HGCalCellOffset(
+    double waferSize, int32_t nFine, int32_t nCoarse, double guardRingOffset_, double mouseBiteCut_) {
+  ncell_[0] = nFine;
+  ncell_[1] = nCoarse;
+  hgcalcell = std::make_unique<HGCalCell>(waferSize, nFine, nCoarse);
+  for (int k = 0; k < 2; ++k) {
+    cellX_[k] = waferSize / (3 * ncell_[k]);
+    cellY_[k] = sqrt3By2_ * cellX_[k];
+    fullArea[k] = 3 * sqrt3By2_ * cellY_[k];
+    std::vector<std::vector<double>> tempOffsetX;
+    std::vector<std::vector<double>> tempOffsetY;
+    for (int j = 0; j < 3; ++j) {
+      if (j == 0) {
```

Comment:
Could you make the code a little more expressive? what does `j=={0,1,2}` represent? Can we have it spelt out so that poeple reading the code could have an idea of what is going on?

Reply:
Added comments explaining the values